actionpack/lib/action_dispatch/integration_test.rb

```diff
@@ -0,0 +1,13 @@
+module ActionDispatch
+  class IntegrationTest < IntegrationTestCase
+    def initialize(*)
+      ActiveSupport::Deprecation.warn(<<-MSG.squish)
+        ActionDispatch::IntegrationTest has been renamed ActionDispatch::IntegrationTestCase.
+        ActionDispatch::IntegrationTest will be removed in Rails 6.0.
+        Please use ActionDispatch::IntegrationTestCase adapter instead.
+      MSG
+
+      super
+    end
```

Comment:
I was chatting with @matthewd and he mentioned that Minitest has since changed their internal naming from using `TestCast` to `Test`.

Can you update this PR to make `IntegrationTestCase` an alias of `IntegrationTest`? That way we don't force users to change, but we can add internal consistency to our test cases?